docs/source/deployment/lambda.mdx

```diff
@@ -0,0 +1,387 @@
+  },
+  "requestContext": {},
+  "body": "{\"operationName\": null, \"variables\": null, \"query\": \"{ hello }\"}"
+}
+```
+
+Now we can use `serverless` to invoke our handler using the query above:
+
+```bash
+serverless invoke local -f graphql -p query.json
+```
+
+Your response should look something like this:
+
+```json
+{
+  "statusCode": 200,
+  "headers": {
+    "cache-control": "no-store",
+    "content-type": "application/json; charset=utf-8",
+    "content-length": "27"
+  },
+  "body": "{\"data\":{\"hello\":\"world\"}}\n"
+}
+```
+
+With everything working locally, we can move on to deployment!
+
+### Deploying
+
+[As we mentioned earlier](#installing-the-serverless-cli), Serverless already has access to your AWS CLI credentials, so to deploy, all you need to do is run the following command:
+
+```bash
+serverless deploy
+```
+
+If successful, `serverless` should output something like this:
+
+```bash
+> serverless deploy
+> Deploying apollo-lambda to stage dev (us-east-1)
+> ✔ Service deployed to stack apollo-lambda-dev (187s)
+> ..............
+> endpoints:
+> POST - https://ujt89xxyn3.execute-api.us-east-1.amazonaws.com/dev/
+> GET - https://ujt89xxyn3.execute-api.us-east-1.amazonaws.com/dev/
+> functions:
+> graphql: apollo-lambda-dev-graphql
+> Monitor all your API routes with Serverless Console: run "serverless --console"
+```
+
+You can now navigate to your endpoints and query your newly hosted server using [Apollo Sandbox](/studio/explorer/sandbox).
+
+#### What does `serverless` do?
+
+First, it builds the functions, zips up the artifacts, and uploads them to a new S3 bucket. Then, it creates a Lambda function with those artifacts and outputs the HTTP endpoint URLs to the console if everything is successful.
+
+### Managing the resulting services
+
+The resulting S3 buckets and Lambda functions are accessible from the [AWS Console](https://console.aws.amazon.com). The AWS Console also lets you view the [IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/getting-started.html) you created earlier.
+
+<!-- cSpell:disable-next-line -->
+
+To find the S3 bucket that Serverless created, search in Amazon's listed services for S3, then look for the name of your bucket (e.g., `apollo-lambda-dev-serverlessdeploymentbucket-1s10e00wvoe5f` is the name of our bucket).
+
+To find the Lambda function that Serverless created, search in Amazon's listed services for `Lambda`. Double-check the region at the top right of the screen if your list of Lambda functions is empty or missing your new function. The default region for Serverless deployments is `us-east-1` (N. Virginia).
+
+If you ever want to remove the S3 bucket or Lambda functions that Serverless created, you can run the following command:
+
+```bash
+ serverless remove
+```
+
+## Customizing HTTP behavior
+
+The `@as-integrations/aws-lambda` package is compatible with Lambda's API Gateway V1 (REST) and V2 (HTTP). However, it does not let you customize HTTP behavior directly.
```

Comment:
It also does not support other AWS products that invoke Lambda functions, like Application Load Balancers.

Reply:
Nice call out, tweaked this to: 

> The `@as-integrations/aws-lambda` package is compatible with Lambda's API Gateway V1 (REST) and V2 (HTTP). However, it does not let you customize HTTP behavior directly or support other AWS products that invoke Lambda functions (e.g., such as Application Load Balancers).